src/yogita/FindMissing.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * 
+ *Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 but exactly one number is missing, 
+you need to write a Java program to find that missing number in an array.
+*/
+
+package yogita;
+
+import java.util.Arrays;
+
+public class FindMissing {
+	void findMissingNumber(int[] array) {
+
+		int[] input = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
```

Comment:
please take looks of this code
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/1031/files